lighthouse-core/config/config.js

```diff
@@ -64,13 +64,18 @@ function validateCategories(categories, audits, groups) {
     return;
   }
 
+  const auditsKeyedById = new Map((audits || []).map(audit =>
+    /** @type {[string, LH.Config.AuditDefn]} */
+    ([audit.implementation.meta.id, audit]))
+  );
```

Comment:
the parens here are confusing - it led me to think `Map` had a `.map` function.

is there an alternative way to balance these parentheses?

Reply:
Oh haha, whoops yeah I'll move the second one down :)